src/structures/Webserver.ts

```diff
@@ -53,7 +54,7 @@ export async function WebServer(server: BaseServer) {
     let str = "";
     const conf = server.config.webserver;
 
-    if (server.cdn.version === req.body.version) str += `server|${conf.address}\n`;
+    if (server.cdn.version === req.body.version || Config.bypassVersionCheck) str += `server|${conf.address}\n`;
```

Comment:
for the Config better use `server.config.bypassVersionCheck` instead of importing config back